### ISSUE TITLE
Applying Ease interpolation to everything

### ISSUE THREAD

**Issue by Toxicoow** [opentoonz/opentoonz#2254]
Hello, i would love to apply Ease In/Out to everything in my project, i would love to know if there a faster way to apply custom ease to everything more quicker than:
>selecting a colon
>Set as Ease in/Out
>Copy past 50 to both
>Next KeyFrame
>Copy past 50 to both
>Next KeyFrame
>Copy past 50 to both
>Next colon
It's time consuming.
![ezgif-4-6af16758cd](https://user-images.githubusercontent.com/25736143/44445699-5742fd00-a5e2-11e8-91b1-9bc7dd2feae8.gif)

**Comment by DarrenTAnims** [comment]
I've been meaning to add a bug report similar to this too (although it might be a feature request). This is especially needed when working with meshes as there's so many columns.

I know there's a context menu option, where you can right click between the keys and change the interpolation type or you can select a frame between keys, then choose from the drop down at the right and click apply. I expected to be able to drag from left to right across multiple keys and then do this once for one of them.

**Comment by DarrenTAnims** [comment]
I guess a 2nd part of this, to make this view easier is to change the context menu option, "Show animated only". This should be renamed to, "Show with keys only" as it shows and of the animatable values that have a key in them. There should then be another option, "Show animated only" that shows any values where there is more than 1 type of value in it.

In the example screen shot above there are lots of columns showing with value 0 in them. They don't change. They don't animate, so you don't want to interpolate or edit them. So it'd be really time-saving to be able to hide them, so that you're only working with actual animated values.

**Comment by RodneyBaker** [comment]
Closing this as a feature request but one with a reasonable workarounds and workflows as well.

Since this issue was opened there have been several additions related to setting interpolations/ease.

FIrstly, we can currently select all animated channels in the Function Editor and in the Timeline/xsheet, Right Click and assign an interpolation.  This alone might fullfill the needs of this Feature request.

Additionally:
Consider that we can also have Preferences open while we are working and adjust the default interpolation on the fly so that what we set in one moment can be different in the next.

Also, rather than setting keys in the first place we might consider the use of expressions, file interpolation and the like to change ease... with relative ease simply by importing the data to channels.

Is there still room for improvement?
Certainly.

Recommend reviewing current workflow of settings interpolations/ease with an eye for refining the process even further.
